application/src/main/java/com/softserve/lv460/application/controller/LocalDeviceController.java

```diff
@@ -0,0 +1,52 @@
+package com.softserve.lv460.application.controller;
+
+import com.softserve.lv460.application.dto.localDevice.LocalDeviceRequest;
+import com.softserve.lv460.application.dto.location.LocationResponse;
+import com.softserve.lv460.application.entity.LocalDevice;
+import com.softserve.lv460.application.service.LocalDeviceService;
+import com.softserve.lv460.application.service.LocationService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/location-device")
+@AllArgsConstructor
+public class LocalDeviceController {
+    private LocalDeviceService localDeviceService;
+    private LocationService locationService;
+
+    @PostMapping("/save")
```

Comment:
rename to "/" instead of "/save". Since it's post method, everyone knows that it is supposed to save new entry